src/Components/Members/index.js

```diff
@@ -34,7 +29,10 @@ function Members(props) {
         resource={'Member'}
         onClick={() => props.history.push('/members/form')}
       />
-      <SharedTable data={members} handleDelete={deleteMemb} editLink={'members/form/'} />
+      {loading && <h1>Loading</h1>}
```

Comment:
i think we can delete the loading state and use the one we have declare in the reducer, or if we are using this one we should delete the other one

Reply:
solved